script.js

```diff
@@ -1,29 +1,42 @@
 //local storage//
-const LOCAL_STORAGE_KEY = "user_data";
-
-//fetching data //
-let promise= fetch("https://geektrust.s3-ap-southeast-1.amazonaws.com/adminui-problem/members.json");
-promise.then((response)=>{
+localStorage.removeItem('userdata');
+const UserData = "userdata";
+
+//fetching data and handling local storage//
+//import APIurl from config.js
+import { APIurl } from "./config.js"; //import api url
+let p= fetch(APIurl);
+p.then((response)=>{
     if(!response.ok)
         throw new Error("Network response is not ok"+response.statusText);
     else
 return response.json();
```

Comment:
what is this code, why haven't you used curly braces, any specific reason

Reply:
because it contains only one statement, so when there is only one statement under if else, the curly braces could be avoided. Its acceptable.